Monika After Story/game/event-handler.rpy

```diff
@@ -446,6 +446,33 @@ init 6 python:
             _pool=_pool
         )
 
+    def mas_protectedShowEVL(
+            ev_label,
+            code,
+            unlock=False,
+            _random=False,
+            _pool=False,
+        ):
+        """
+        Shows an event given label and code.
+
+        Does checking if the actions should happen
+        IN:
+            ev_label - label of event to show
+            code - string code of the db this ev_label belongs to
+            unlock - True if we want to unlock this Event
+                (Default: False)
+            _random - True if we want to random this event
+                (Default: False)
+            _pool - True if we want to random thsi event
+                (Default: False)
+        """
+        store.mas_showEvent(
```

Comment:
better to call `mas_showEVL` from here so dont have to copy the `mas_all_ev_db_map` code.